js/widgets/status.js

```diff
@@ -98,12 +98,12 @@ class StatusMatrix {
             }
             cell.style.width = "212.5px";
             this.widgetWindow.onmaximize = () => {
-                this.isMaximized = !(this.isMaximized);
-                console.debug("Maximized " + this.isMaximized);
+                this.isMaximized = !this.isMaximized;
                 cell.style.width = "100vw";
                 cell.style.paddingLeft = "30px";
-                cell.style.fontSize = Math.floor(this._cellScale * StatusMatrix.FONTSCALEFACTOR)*0.90 + "%";
-                if(!(this.isMaximized)){
+                cell.style.fontSize =
+                    Math.floor(this._cellScale * StatusMatrix.FONTSCALEFACTOR) * 0.9 + "%";
```

Comment:
Status is different widget afaik, why is this file a part of this PR?

Sorry, I missed this file previously. But please do tell me If I have missed something and what has this file to do with maximizing musickeyboard widget in particular.

Reply:
Yeah, status is a different widget. Actually, after #2838 was merged, as always I deleted that branch and then realized that I left debug statements (that I used for #2838) in the code and did not Prettify the code. So just to rectify that, I did not feel that a different PR would be needed. Thus, I committed those changes first, so this could be rectified.